### PR TITLE
feat: digital business card

### PR DESCRIPTION
**Pull request by amanji**
Issue #: bcgov/entity#18284 + bcgov/entity#18394

Description of changes: See PR #552 + PR #562

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
I think it would have been preferable to merge from the feature branch to main but this is OK. I'll close #560.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-566-6ks4rfnd.web.app

**Review by severinbeauvais — APPROVED** [review]

File: src/components/DigitalCredentials/dialogs/ReplaceCredentialErrorDialog.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import { Getter } from 'pinia-class'
+import { ContactInfo } from '@/components/common'
+import { useRootStore } from '@/stores'
+
+  @Component({
+    components: { ContactInfo }
+  })
+export default class ReplaceCredentialErrorDialog extends Vue {
+    // Getter to check if logged in user is Staff.
+    @Getter(useRootStore) isRoleStaff!: boolean
```
Comment (JazzarKarim) [inline src/components/DigitalCredentials/dialogs/ReplaceCredentialErrorDialog.vue:54]: Akiff, super small thing. Can we please fixing the spacing here (2 spaces)?

**Review by JazzarKarim — APPROVED** [review]
LGTM! 👍

File: src/enums/webSocketEvents.ts
Comment (argush3) [inline src/enums/webSocketEvents.ts:1]: Can this be removed now that we aren't using websockets anymore?

File: src/enums/webSocketTopics.ts
Comment (argush3) [inline src/enums/webSocketTopics.ts:1]: Can this be removed now that we aren't using websockets anymore?

File: src/enums/index.ts [hunk 2]
```diff
@@ -18,6 +18,8 @@ export * from './routes'
 export * from './saveErrorReasons'
 export * from './digitalCredentialTypes'
 export * from './affiliationInvitation'
+export * from './webSocketEvents'
+export * from './webSocketTopics'
```
Comment (argush3) [inline src/enums/index.ts:22]: Can this be removed now that we aren't using websockets anymore?

File: src/enums/webSocketEvents.ts
Comment (amanji) [reply src/enums/webSocketEvents.ts:1]: Yeah we should. We can add it back later

**Review by severinbeauvais — APPROVED** [review]

**Review by argush3 — APPROVED** [review]

**Review by JazzarKarim — APPROVED** [review]

**Pull request merged**